udata/harvest/backends/dcat.py

```diff
@@ -355,7 +353,8 @@ def walk_graph(self, url: str, fmt: str) -> Generator[tuple[int, Graph], None, N
         See https://github.com/SEMICeu/iso-19139-to-dcat-ap for more information on the XSLT.
         """
         # Load XSLT
-        xsl = ET.fromstring(self.get(self.XSL_URL).content, parser=SAFE_PARSER)
+        xsl_url = current_app.config.get("HARVEST_ISO19139_XSL_URL")
```

Comment:
```suggestion
        xsl_url = current_app.config["HARVEST_ISO19139_XSL_URL"]
```
Either that, or guard against None some other way.

Reply:
Yes, which is why I suggest using `[]` instead of `get`, to make it explicit you expect to always get a value.